clang-tools-extra/templight/templight_driver.cpp

```diff
@@ -365,7 +365,7 @@ static int ExecuteTemplightInvocation(CompilerInstance *Clang) {
   // This should happen AFTER plugins have been loaded!
   if (!Clang->getFrontendOpts().LLVMArgs.empty()) {
     unsigned NumArgs = Clang->getFrontendOpts().LLVMArgs.size();
-    auto Args = std::make_unique<const char *[]>(NumArgs + 2);
+    auto Args = make_unique<const char *[]>(NumArgs + 2);
```

Comment:
why is this change being made from the mainline import?

Reply:
The mainline is set to clang 8. For clang 9, they're now using c++11 so there's no longer a need for their old custom make_unique calls.